server/shared/src/main/scala/org/http4s/server/middleware/Throttle.scala

```diff
@@ -80,7 +80,7 @@ object Throttle {
       override def takeToken: F[TokenAvailability] =
         for {
           values <- counter.access
-          previousTokens -> previousTime -> setter = values
+          ((previousTokens, previousTime), setter) = values
```

Comment:
Huh, how is this related to JS? The compiler frontends should be the same, otherwise this is a bug 😂

Reply:
Nahh, ignore the log message: it's an issue with both scala 2.12 and JS!

Reply:
If it's specific to JS, that's still a bug :) does it happen on 2.12 JVM?

Reply:
Apparently so, as it was failing on the `rootJVM` job compiling against `2.12.x`

Reply:
This is the offending job for 2.12 JVM: https://github.com/http4s/http4s/runs/6951228394?check_suite_focus=true